src/app/github-landing-page/github-landing-page.component.html

```diff
@@ -0,0 +1,95 @@
+<!--
+  ~    Copyright 2024 OICR, UCSC
+  ~
+  ~    Licensed under the Apache License, Version 2.0 (the "License");
+  ~    you may not use this file except in compliance with the License.
+  ~    You may obtain a copy of the License at
+  ~
+  ~        http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~    Unless required by applicable law or agreed to in writing, software
+  ~    distributed under the License is distributed on an "AS IS" BASIS,
+  ~    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~    See the License for the specific language governing permissions and
+  ~    limitations under the License.
+  -->
+
+<div class="overflow-hidden">
+  <div class="mb-5 w-100 landing-page-container">
+    <div class="header-container w-100">
+      <h3>Please wait while we register your GitHub repository. This may take a few minutes.</h3>
+    </div>
+    <hr />
+    <h3 class="close-tab-message">You can close this tab and return to Dockstore on the original tab.</h3>
+
+    <div class="w-100 cards-container" fxLayout="row" fxLayoutAlign="space-evenly stretch">
+      <mat-card class="landing-page-card">
+        <mat-card-header>
+          <mat-card-title>Do you have a .dockstore.yml file in your repo?</mat-card-title>
```

Comment:
Here and line 33 it's repo, line 20 it's repository. We should be consistent -- I lean towards `repository`; it's what the GitHub UI uses.